tests/rados/test_track_availibility_status.py

```diff
@@ -23,385 +25,647 @@
+            log_msgs = [
+                "'mgrstat pool_availability'",
+                "'mgrstat calc_pool_availability: Adding pool'",
+                "'Deleting pool'",
+            ]
+            for log_line in log_msgs:
+                msg_info = (
+                    f"Case2:Verification of the log message -{log_line} in the osd logs"
+                )
+                log.info(msg_info)
+                if (
+                    rados_object.lookup_log_message(
+                        init_time=init_time,
+                        end_time=end_time,
+                        daemon_type="mon",
+                        daemon_id=leader_mon,
+                        search_string=log_line,
+                    )
+                    is False
+                ):
+                    msg_error = (
+                        f"Case2:During the verification of the track availability tests the {log_line} not exists in "
+                        f"the logs"
+                    )
+                    log.error(msg_error)
+                    return 1
+
+            log.info(
+                "===  Case 2: Verification of track availability messages in the logs are completed==="
+            )
+            log.info(
+                "==== Case 2: Verification of the score values displayed on multiple pools and verification of the "
+                "logs completed ===="
+            )
+        if "case3" in config.get("case_to_run"):
+            log.info(
+                "=== Case 3: Verification of track availability tests during recovery state==="
+            )
+            assert create_data_pool(mon_obj, rados_object, replicated_config, pool_name)
+            method_should_succeed(
+                wait_for_clean_pg_sets, rados_object, test_pool=pool_name, timeout=1800
+            )
+            acting_pg_set = rados_object.get_pg_acting_set(pool_name=pool_name)
+            if not rados_object.change_osd_state(
+                action="stop", target=acting_pg_set[0]
+            ):
+                msg_error = f"Case3:Unable to stop the OSD : {acting_pg_set[0]}"
+                log.error(msg_error)
+                raise Exception("Exception occurred while stopping the OSD ")
+
+            rados_object.bench_write(
+                pool_name=pool_name, byte_size="5K", max_objs=10000
+            )
+            if not rados_object.change_osd_state(
+                action="start", target=acting_pg_set[0]
+            ):
+                msg_error = f"case3:Unable to stop the OSD : {acting_pg_set[0]}"
+                log.error(msg_error)
+                raise Exception("Exception occurred while stopping the OSD")
+
+            timeout = datetime.timedelta(minutes=15)
+            start_time = datetime.datetime.now()
+            recovering_found = False
+
+            while datetime.datetime.now() - start_time < timeout:
+                out_put = rados_object.run_ceph_command(cmd="ceph -s")
```

Comment:
Collect the state for the Individual PG which we are tracking, instead of ceph status